spec/features/movies_feature_spec.rb

```diff
@@ -25,420 +25,417 @@
       { name: "Foodoo", url: "https://www.foodoo.com/search?searchString=Fake", pay_model: "rent" }
     ]}
 
-    describe "movie show page functionality" do
+    describe 'move show page' do
       before do
-        allow(StreamingServiceProviderDataService).to receive(:get_providers).and_return(streaming_service_providers)
+        allow(StreamingServiceProviderDataService)
+          .to receive(:get_providers)
+          .and_return(streaming_service_providers)
```

Comment:
Really the only change in this file is to add one parent describe block so that the stubbing of StreamingServiceProviderDataService only needs to happen once. Prolly best to view with whitespace off.